content/id/docs/concepts/workloads/pods/disruptions.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Disrupsi
+content_template: templates/concept
+weight: 60
+---
+
+{{% capture overview %}}
+Petunjuk ini diperuntukkan bagi pemilik aplikasi yang ingin membuat aplikasi yang memiliki ketersediaan yang tinggi, sehingga dibutuhkan untuk mengerti jenis-jenis Gangguan yang dapat terjadi pada Pods.
+
+Petunjuk ini juga diperuntukkan bagi administrator kluster yang ingin melakukan berbagai tindakan otomasi pada kluster, seperti pembaruan dan *autoscaling* kluster.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Disrupsi yang Disengaja dan Tidak Disengaja
+
+Pods tidak akan terhapus hingga sesuatu (orang maupun *controller*) menghancurkan mereka atau ada kesalahan perangkat keras maupun perangkat lunak yang tidak dapat dihindari.
```

Comment:
prefer singular over plural for imported word / kubernetes terms: Pod (not Pods)

or do "Semua/banyak Pod" if you want to express plural